index.js

```diff
@@ -3,83 +3,48 @@
 
 var Promise = require('ember-cli/lib/ext/promise');
 
-var chalk = require('chalk');
-var blue  = chalk.blue;
-var red   = chalk.red;
-
-var validateConfig = require('./lib/utilities/validate-config');
+var DeployPluginBase = require('ember-cli-deploy-plugin');
```

Comment:
Hey @lukemelia, I may be having a massive brain fart but where does `ember-cli-deploy-plugin` come from?  I can't see it in the ember-cli/ember-cli-deploy#180 which is where I would have expected it to be.